addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -1730,7 +1752,7 @@
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'base',
-                    'minus_report_line_ids': [ref('l10n_de.tax_report_de_tag_68')],
+                    'minus_report_line_ids': [ref('l10n_de.tax_report_de_tag_60')],
```

Comment:
I don't get why this is a minus report line:
- it's for invoice line, not refund
- the factor percent is positive
Does it gives the expected result in the report ?

Reply:
It's probably not right, but we don't know what exactly should be the right tax (probaly not this tax at all that should be used here, but who knows, we keep the same behaviour as before...), so we won't fix it in this PR. (asked tsb)